docarray/array/storage/base/backend.py

```diff
@@ -25,10 +25,13 @@ def _get_storage_infos(self) -> Optional[Dict]:
     def _map_id(self, _id: str) -> str:
         return _id
 
+    def _map_column(self, value, col_type) -> str:
+        return self.TYPE_MAP[col_type][1](value)
```

Comment:
If it was a namedtuple we could have something like 
```suggestion
        return self.TYPE_MAP[col_type].type(value)
```